cache/test_cache.py

```diff
@@ -2485,4 +2485,58 @@ def test(self, name, client_id1, client_id2, method):
         self.check_second_request(client=client2, method=method, tr1="X-Token1", tr2="X-Token2")
 
 
+@marks.parameterize_class(
+    [
+        {"name": "Http", "clients": [DEPROXY_CLIENT]},
+        {"name": "H2", "clients": [DEPROXY_CLIENT_H2]},
+    ]
+)
+class TestCacheAgeInResponse(tester.TempestaTest):
+    tempesta = {
+        "config": """
+listen 80;
+listen 443 proto=h2;
+
+cache 2;
+cache_fulfill * *;
+
+server ${server_ip}:8000;
+
+tls_certificate ${tempesta_workdir}/tempesta.crt;
+tls_certificate_key ${tempesta_workdir}/tempesta.key;
+tls_match_any_server_name;
+""",
+    }
+
+    backends = [DEPROXY_SERVER]
+
+    def test(self):
+        self.start_all_services()
+
+        srv: StaticDeproxyServer = self.get_server("deproxy")
+        srv.set_response(
+            "HTTP/1.1 200 OK\r\n"
+            + "Connection: keep-alive\r\n"
+            + "Content-Length: 13\r\n"
+            + "Content-Type: text/html\r\n"
+            + f"Date: {HttpMessage.date_time_string()}\r\n"
+            + "age: 333\r\n"
+            + "\r\n"
+            + "<html></html>"
+        )
+
+        client = self.get_client("deproxy")
+
+        request = client.create_request(
+            method="GET",
+            uri="/",
+            headers=[],
+        )
+
+        client.send_request(request, "200")
+        self.assertEqual(client.last_response.headers.get("age"), "333")
+
+        client.send_request(request, "200")
+        self.assertEqual(client.last_response.headers.get("age"), "333")
```

Comment:
Please add the checks for the second response.  Did you receive this response from Tempesta or Sverer? Why?